Mathlib/GroupTheory/GroupAction/BigOperators.lean

```diff
@@ -67,4 +68,9 @@ theorem Finset.smul_prod {r : α} {f : γ → β} {s : Finset γ} :
   map_prod (MulDistribMulAction.toMonoidHom β r) f s
 #align finset.smul_prod Finset.smul_prod
 
+theorem smul_finprod [Finite γ] {f : γ → β} (r : α) :
```

Comment:
I think you want to state this for `γ : Sort*`?

Reply:
ha ha that might be the case but then I can't prove it :-)

Reply:
I think you then have to use `cases nonempty_fintype (PLift γ)`, and you can probably work out the rest by looking for other uses of that pattern.